config/deploy.rb

```diff
@@ -31,7 +31,7 @@
   desc 'Link settings.php'
   task :link_settings do
     on roles(:app) do |_host|
-      execute "cd #{release_path}/sites/#{fetch(:drupal_site)} && ln -sf #{fetch(:drupal_settings)} settings.php"
+      execute "cd #{release_path}/sites/#{fetch(:drupal_site)} && cp #{fetch(:drupal_settings)} settings.php"
```

Comment:
I am terrible at Drupal, but could this risk not being able to update the site settings via the UI?

Reply:
that makes sense.